### ISSUE TITLE
Suggestion - 

### ISSUE THREAD

**Issue by Luckyradkid1** [StargateMC/IssueTracker#220]
## What is your suggestion?
to add a supergate maybe

>Why is this important?
because it cannon, and would allow players to travel to another point in space at expense of lots of resources and time quickly

## How will this benefit players?
it would allow players to quickly travel to another point in space without the wait time of the warp
>Why is this important?
because it could be very helpful in a number of ways, im not going to list them right now

## How will this change for players?
it could help them in numbers of ways

>Why is this important?
because sometimes you need to quickly leave your area or need to do something in another galaxy quickly.

## Do you consider this suggestion a:

- [x] Must (Important to keep players happy)
- [x] Should (Beneficial to players, but can do without)
- [x] Could (Beneficial to players, there is no important reason to do it
i would say it is a should, it would benifit players a lot, but this server dosnt need it to work :)

>extra things?
supergate would be expensive and lots of blocks, plus it should only work with black holes in the system on both ends, and after the activation, im not sure how you would make the ship go through, maybe after 5 or 10 seconds the ship goes to whereever the next supergate is. the person would also have to take into acount the unstable event horizon? so to not build their supergate facing directly at their ship, so it wouldnt get destroyed, and the supergate, once built completly would not move with the ship, so if i were to warp to another planet wiht a supergate next to me it wouldnt come wiht me.

**Comment by Joey4213** [comment]
Its a bit of a gray area what you are suggesting but I will see if its even remotely possible

**Comment by draksterau** [comment]
The way I see supergates functioning would be:

- AR Ships can see them from orbit.
- Animations would show you travelling through a gate when in FTL.
- FTL would only survive as long as the gate transition, at which point you'd be on the other end.


They would not be virtual objects, and players wouldnt be able to transition through them without an AR vessel.

The primary purpose of them would be to instantly transport a star-ship over vast distances.


Thoughts?

**Comment by Joey4213** [comment]
sounds good to me

**Comment by TheRedCreations** [comment]
> 
>    - AR Ships can see them from orbit.
>    - Animations would show you travelling through a gate when in FTL.
>    - FTL would only survive as long as the gate transition, at which point you'd be on the other end.
> 
I like your Idea

**Comment by Luckyradkid1** [comment]
yes that sounds great to me :) only thing how would you build one? and put it in orbit? maybe the same way to build a space station?

**Comment by TheRedCreations** [comment]
i suggest to send a supergate via sattelite rocket into space. and a supergate works only in BH systems

**Comment by Luckyradkid1** [comment]
yeah

**Comment by Luckyradkid1** [comment]
um maybe you need like i dunno 10 or 15 sattelite bays in total to get it in space? like, maybe you build one of the "ships" (as they call the individual linked things) things and send it into space, then in orbit u only see one of the ships, then i can put another ship in the sattelite bay and send it back to space and then you would see two ships, or could just put them all in at once, but once you keep doing that and its complete then you can travel to another supergate?
